src/stories/containers/RecognizedDelegates/KeyStats/KeyDescriptionCards/VisualizationCard/VisualizationCard.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import KeyStatsCard from '../../../GenericDelegateCard';
+import LegendItem from './LegendItem';
+import { RelativeDelegateBar } from './RelativeDelegateBar';
+
+interface Props {
+  percent: number;
+  totalDai: number;
+}
+
+const VisualizationCard: React.FC<Props> = ({ percent }) => (
+  <ExtendedKeyStatsCard>
+    <Legend>
+      <LegendItem color="#ECF1F3" description="Other Expenses" />
+      <LegendItem color="#447AFB" description="Recognized Delegates (21)" />
+    </Legend>
+    <ContainerBar>
+      <BarPercent>
+        <RelativeDelegateBar otherExpenses={90} recognizedDelegates={15} />
+      </BarPercent>
+      <BarDescription>
+        <NumberPercent>{`${percent}%`}</NumberPercent>
+        <Total>
+          19, 614, 105
```

Comment:
remember that this will be a number that requires to be formatted, use the helper previously mentioned